source/_components/sensor.envirophat.markdown

```diff
@@ -0,0 +1,132 @@
+---
+layout: page
+title: "Enviro pHAT"
+description: "Instructions how to integrate the Enviro pHAT within Home Assistant."
+date: 2017-05-03 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: raspberry-pi.png
+ha_category: Sensor
+ha_iot_class: "Local Push"
```

Comment:
It was merged but not released yet, what should happen, wait for the actual release, or how to know what the `ha_version` value should be?